src/app/helptext/storage/volumes/datasets/dataset-unlock.ts

```diff
@@ -0,0 +1,38 @@
+import {Validators} from '@angular/forms';
+import { T } from '../../../../translate-marker';
+
+export default {
+    dataset_unlock_title: T('Unlock Datasets'),
+    unlock_key_file_placeholder: T('Unlock with Key file'),
+    unlock_key_file_tooltip: T('Use an exported encryption key file to unlock datasets.'),
+    unlock_children_placeholder: T('Unlock Children'),
+    unlock_children_tooltip: T('Also unlock any encrypted dataset stored within this dataset.'),
+    restart_services_placeholder: T('Restart attached VMs, Jails and Services'),
+    restart_services_tooltip: T('Restarting system services after unlocking the dataset makes \
+     the information contained within the dataset available to the rest of the system.'),
+    upload_key_file_placeholder: T('Upload Key file'),
+    upload_key_file_tooltip: T('Browse to the exported key file that can be used to unlock this \
+     dataset.'),
+    dataset_key_placeholder: T('Dataset Key'),
+    dataset_key_tooltip: T(''),
```

Comment:
@freenas/docs-team can I get a tooltip suggestion for this one?  I presume it could be something similar to how dataset_passphrase is, it is a text area that the user can copy and paste the key into

Reply:
I couldn't find this field when build testing. I'm updating my system and will look for it again.

Reply:
It is hidden by default, you have to uncheck inherit, then set the key type instead of passphrase